docs/dev/configuration.rst

```diff
@@ -0,0 +1,43 @@
+Configuration module
+=====================
+
+Front-end configuration
+------------
+
+It is possible to perform simple configuration of the MO frontend using the
+configuation file found in ``backend/mora/user_settings.json`` or in the file
```

Comment:
If we retain this, we should add, and mention, a `user_settings-example.json` just as we have `mora-example.json`.